README.md

```diff
@@ -147,18 +147,9 @@ so it is reasonable to retire BOMs for lines older than that.
 
 ## Releasing
 
-`release:prepare` only runs basic tests about plugin versions, not the full PCT.
-Therefore be sure to check [commit status](https://github.com/jenkinsci/bom/commits/master)
-to ensure that CI builds have passed before cutting a release.
-
-Due to a misconfiguration in Incrementals tooling (JENKINS-58641),
-currently after every release you must manually run
-
-```bash
-mvn -f sample-plugin incrementals:reincrementalify
-```
-
-Commit and push the result to fix the branch build.
+Automatic given [JEP-229](https://jenkins.io/jep/229) when PRs matching certain label patterns are merged.
+For the common case that only lots of `dependencies` PRs have been merged,
```

Comment:
Given that #505 was an `enhancement`, I would expect merging _this_ PR to automatically trigger a release after a suitable delay (assuming no ci.jenkins.io outage of course).